lib/repo-audit/checks/at_least_one_file_check.rb

```diff
@@ -2,7 +2,7 @@ module RepoAudit::Checks
   class AtLeastOneFileCheck < BaseCheck
     include RepoAudit::RepositoryContent
 
-    register_check :at_least_one_file_check
+    register_check :at_least_one_file
```

Comment:
Yeah, @zheileman and I had a conversation about this on Friday. We disagree about where the configuration should live. I argued for the yaml file having a check like 'has_continuous_integration_configured' - i.e. the configuration of what that means lives in the check class.
He prefers the yaml file itself to embody the configuration of how to tell whether CI is set up or not. 
Both arguments have merit, so we left it done his way.